bin/setup

```diff
@@ -12,6 +12,7 @@ bundle check || bundle install
 
 # Set up database and add any development seed data
 bin/rake dev:prime
+bin/rake db:seed
```

Comment:
@paulcsmith, Is this how I would add `rake db:seed` to my `bin/setup` file?

Reply:
Yep!